GMPWrapper/scripts/convertMarlinSteeringToGaudi.py

```diff
@@ -124,14 +124,30 @@ def createFooter(lines, glob):
   lines.append("                EvtSel = 'NONE',")
   lines.append("                EvtMax   = 10,")
   lines.append("                ExtSvc = [evtsvc],")
-  lines.append("                OutputLevel=%s" % glob.get("Verbosity", "DEBUG"))
+  lines.append("                OutputLevel=%s" % verbosityTranslator(glob.get("Verbosity", "DEBUG")))
   lines.append("              )\n")
 
 
+def verbosityTranslator(marlinLogLevel):
+  """Translate the verbosity level from Marlin to a Gaudi level.
+
+  Marlin log level can end with numbers, e.g., MESSAGE4
+  """
+  logLevelDict = {'DEBUG': 'DEBUG',
+                  'MESSAGE': 'INFO',
+                  'WARNING': 'WARNING',
+                  'ERROR': 'ERROR',
+                  'SILENT': 'FATAL',
```

Comment:
SILENT = FATAL? this seems contradictory to me.

Reply:
There is no equivalent for SILENT, so printing nothing unless FATAL seems to be the best fit.